### ISSUE TITLE
[Data@TAMU] Search page

### ISSUE THREAD

**Issue by devangm** [TAMULib/data-dspace-angular#15]
**URL:** https://data-pre.library.tamu.edu/search?spc.page=1&spc.sf=dc.date.accessioned&spc.sd=DESC
**WCAG 2.1 AA compliant? :** No
**Violation:** 4.1.2 Name, Role, Value (A)
**Issues:** Broken ARIA menu
**Notes:** There is a menubar with no items within it: `<div _ngcontent-dspace-angular-c2948902503="" role="menubar" class="dso-edit-menu d-flex">`

**Comment by tamu-sad-iii** [comment]
I am unable to find the Broken ARIA menu on the provided URL.

<img width="1920" height="1008" alt="Image" src="https://github.com/user-attachments/assets/515b723a-a305-4a9c-b1a8-f1f1f36cd074" />
